node.js

```diff
@@ -48,10 +48,10 @@ module.exports = {
       plugins: ['flowtype'],
       extends: ['plugin:flowtype/recommended', 'prettier/flowtype'],
       settings: {
-        'import/extensions': extensions.ALL,
+        'import/extensions': extensions.JS,
```

Comment:
It seems that we would still need to add extensions.TS here, because otherwise we do not support importing .ts files from .js.

Reply:
I don't think we should support that, nor it is a thing that you easily spot in the wild? Maybe I'm wrong, but I've never seen people importing `.ts` from `.js` files, have seen the opposite though

Reply:
IMO JS/TS interoperability should support both directions (imports JS from TS, and import TS from JS). One scenario that comes to my mind is progressive migration from pure JS to TS, when you update components one by one.

Reply:
When we migrated Jest iteratively to TS, we needed to rework all imports from a converted file to be TS as well, but maybe that was because of specific config setup, don't remember. Anyway we support both scenarios now :)